test/infamy/restconf.py

```diff
@@ -207,7 +207,13 @@ def get_datastore(self, datastore="operational", path="", parse=True):
             dspath = f"{dspath}/{path}"
 
         url = f"{self.restconf_url}{dspath}"
-        return self._get_raw(url, parse)
+        try:
+            return self._get_raw(url, parse)
+        except Exception as e:
```

Comment:
```suggestion
        except requests.exceptions.HTTPError as e:
```
If something else goes wrong, I think we risk triggering an `AttributeError` when trying to access `e.response` below, loosing the original exception information.

Reply:
True, thanks!

I have more fixes coming. QoS has stopped working as well, for some reason.